src/js/users/components/__tests__/Create.test.jsx

```diff
@@ -22,80 +51,46 @@ describe("<CreateUser />", () => {
             password: "",
             handle: ""
         };
+        history = createBrowserHistory();
     });
 
-    it("should render", () => {
-        const wrapper = shallow(<CreateUser {...props} />);
-        expect(wrapper).toMatchSnapshot();
+    it("should render correctly when show=true", () => {
+        renderWithRouter(<CreateUser {...props} />, state, history);
+        expect(screen.getByText("Create User")).toBeInTheDocument();
+        expect(screen.getByText("Username")).toBeInTheDocument();
+        expect(screen.getByText("Password")).toBeInTheDocument();
+        expect(screen.getByText("Force user to reset password on login"));
+        expect(screen.getByRole("button", { name: "Save" })).toBeInTheDocument();
     });
 
-    it("should render with error", () => {
-        const wrapper = shallow(<CreateUser {...props} />);
-        wrapper.setState({
-            errorPassword: "Password too short"
-        });
-        expect(wrapper).toMatchSnapshot();
+    it("should render correct error messages", () => {
+        renderWithRouter(<CreateUser {...props} />, state, history);
+        const saveButton = screen.getByRole("button", { name: "Save" });
+        saveButton.click();
+        expect(screen.getByText("Please specify a username")).toBeInTheDocument();
+        expect(screen.getByText("Passwords must contain at least 8 characters")).toBeInTheDocument();
+        expect(props.onCreate).not.toHaveBeenCalled();
```

Comment:
Example of making test more readable.
```suggestion
        renderWithRouter(<CreateUser {...props} />, state, history);
        
        userEvent.click(screen.getByRole("button", { name: "Save" }));
        
        expect(screen.getByText("Please specify a username")).toBeInTheDocument();
        expect(screen.getByText("Passwords must contain at least 8 characters")).toBeInTheDocument();
        expect(props.onCreate).not.toHaveBeenCalled();
```